### PR TITLE
[14.0][ADD] epa_sale_crm_custom: add new module

### PR DESCRIPTION
**Pull request by kaynnan**
This pull request introduces a new module, `EPA Sale CRM Custom`, which extends the functionality of Odoo's CRM, Sales Order, and Project Category modules. The key features include automatic codename generation for leads, criteria field validation, analytic account linking, and attachment management in Sales Orders.

### Module Introduction and Features
* **README file**: Added a comprehensive description of the module, its key features, configuration steps, and usage instructions.

### Module Initialization and Manifest
* **Initialization**: Imported `models` and `wizard` in the `__init__.py` file.
* **Manifest**: Created the `__manifest__.py` file with metadata, dependencies, and data files required for the module.

### Data and Configuration
* **Sequence Configuration**: Added XML configuration for custom sequences for sale orders and CRM leads.

### Internationalization
* **Translation Template**: Added a `.pot` file for translations, including various field descriptions and messages.

[HT00800]

cc @marcelsavegnago @WesleyOliveira98 

image

- [x] Sequence in sale.order
- [x] Sequence in crm.leader
- [x] Wizard possibility Create/Select to Analytic Account before New Quotation
- [x] Project assumes the same name as the sale.order, removing the SO prefix
- [x] Changes to include Partial CRM (related to Rodrigo Rameh modifications)

### CODE REVIEW

**Review by WesleyOliveira98 — APPROVED** [review]
LGTM

**Comment by marcelsavegnago** [comment]
podemos mesclar ?

**Comment by kaynnan** [comment]
> podemos mesclar ?

Por momento acredito que sim, estamos aguardando o cliente retornar das ferias para apresentar, mas qualquer mudança abro uma nova PR, por momento, mantem somente na Staging

File: epa_sale_crm_custom/data/sequence.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <record id="seq_epa_sale_order_custom_sale" model="ir.sequence">
         <field name="name">Custom Sale Order Sequence (in epa_sale_crm_custom)</field>
         <field name="code">sale.order</field>
-        <field name="prefix">SO-%(year)s-%(month)s-</field>
+        <field name="prefix">%(y)s-</field>
```
Comment (WesleyOliveira98) [inline epa_sale_crm_custom/data/sequence.xml]: Aqui pode manter o `SO-%(y)s`

**Review by WesleyOliveira98 — CHANGES_REQUESTED** [review]

**Review by WesleyOliveira98 — APPROVED** [review]
LGTM

**Review by WesleyOliveira98 — APPROVED** [review]
LGTM

**Pull request merged**